modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -177,6 +177,12 @@ else if (key.contains("objectRelationshipName")) {
 			paths.remove(key);
 		}
 
+		if (!_objectDefinition.isEnableObjectEntryVersioning()) {
+			objectDefinitionSchemaProperties.remove("systemProperties");
+
+			schemas.remove("SystemProperties");
+		}
+
 		if (!_objectDefinition.isEnableCategorization()) {
```

Comment:
This condition is not sorted. Can you sort it, please?
`if (!_objectDefinition.isEnableObjectEntryVersioning()) {
`
`if (!_objectDefinition.isEnableCategorization()) {
`